pikaur/args.py

```diff
@@ -82,6 +82,7 @@ def get_pikaur_bool_opts() -> ArgSchema:
 
 def get_pikaur_str_opts() -> ArgSchema:
     return [
+        (None, 'build-gpgdir', None),
```

Comment:
you need to add `PikaurConfig().build.GpgDir.get_str()` here instead of `build.py`